pom.xml

```diff
@@ -28,7 +28,7 @@
     <groupId>org.xwiki.commons</groupId>
     <artifactId>xwiki-commons-pom</artifactId>
     <!-- The application meeting use the new Mail Sender API which bundled by default in XWiki Enterprise since 6.2M1 -->
-    <version>6.2-milestone-1</version>
+    <version>6.4-milestone-3</version>
```

Comment:
Lyes you need to think about what you're doing. Setting the version as you mentioned means nobody using a version older than 6.4M3 can use the modified meeting app!